src/components/missions/Mission.js

```diff
@@ -1,11 +1,48 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
+import { toggleMissionDispatcher } from '../../redux/missions/missions';
 
 export default function Mission({ mission }) {
+  const dispatch = useDispatch();
+
+  const handleJoinMission = (id) => {
+    dispatch(toggleMissionDispatcher(id));
+  };
+  const handleLeaveMission = (id) => {
+    dispatch(toggleMissionDispatcher(id));
+  };
+
+  const status = mission.status ? (
+    <div className="active-mb">Active Member</div>
+  ) : (
+    <div className="no-active-mb"> NOT A MEMBER</div>
+  );
```

Comment:
The way you render the status is working perfectly fine but I suggest using the React conditional rendering syntax
since it's in the requirements.
`mission. status && <div className="active-mb">Active Member</div> `

Reply:
Ok thank you, I'll do it